version/version.go

```diff
@@ -4,11 +4,11 @@ import "fmt"
 
 const (
 	// VersionMajor is for an API incompatible changes
-	VersionMajor = 4
+	VersionMajor = 5
 	// VersionMinor is for functionality in a backwards-compatible manner
 	VersionMinor = 0
 	// VersionPatch is for backwards-compatible bug fixes
-	VersionPatch = 2
+	VersionPatch = 1
```

Comment:
Should this be 0 for 5.0.0?

Reply:
Compare https://github.com/containers/image/pull/735/commits/194eb017e848758aeccedd725c4c1a49fb2f7f67 and https://github.com/containers/image/pull/735/commits/a40e6b800d77a0603389ab5f1a78e430601b2975 .

Reply:
Doh.  Missed that it was multiple commits.  Looks good.